pkg/scheduler/framework/plugins/nodevolumelimits/utils.go

```diff
@@ -40,21 +40,15 @@ func isCSIMigrationOn(csiNode *storagev1.CSINode, pluginName string) bool {
 	// along with the plugin-specific one
 	switch pluginName {
 	case csilibplugins.AWSEBSInTreePluginName:
-		if !utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationAWS) {
-			return false
-		}
+		return true
```

Comment:
is it suppose to be `true` or `false`? Does it have to return at all? Today the feature gate is locked to `true`, meaning we will not enter this if statement and will just go forward executing the rest of the function without returning from here

Reply:
Yes, it's supposed to be `true`,  I think it's the same on https://github.com/kubernetes/kubernetes/pull/115838/files

Reply:
cc @torredil @dims ^

Reply:
looks like https://github.com/kubernetes/kubernetes/pull/115838/files#diff-6901af4ac953694f828dd8bac6f624717b2f622a143f293b9258a46f9a2a559fR43 locked it to true

that seems correct... if asked, AWS migration *is* on

Reply:
Let's ask the folks who know best... @jsafrane @xing-yang What should it be here?

Reply:
This should be set to `true`.

Reply:
is it not working as it should be working on 1.26 where the feature gate is locked to true?

Reply:
I think it is. If there needs a double-checked, Let's ask someone who knows... 

/cc @torredil @jsafrane thanks

Reply:
> Yes, this code chunk is correct. `isCSIMigrationOn()` should always return `true` for AzureDisk and GCE PD, because their CSI migration was GA and locked and they're being removed here.

I am asking about 1.26 release. Was it suppose to return true when `utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationAWS)` is true?

Reply:
I understand that the unsupported skews are not supported. But would it make sense to keep this logic for longer? What the worst can happen if we are running with older than supported nodes and how easy it will be discovered by the end user?

Reply:
`CSIMigrationGCE` is GA + lock to true in 1.25: https://github.com/kubernetes/kubernetes/blob/release-1.25/pkg/features/kube_features.go#L923